wherehows-frontend/application.env

```diff
@@ -0,0 +1,37 @@
+# Directory containing kerberos config files
+WHZ_KRB5_DIR="/dir/to/krb5conf"
```

Comment:
For frontend, don't require this config, see wherehows-frontend/conf/application.conf